.github/workflows/testing.yml

```diff
@@ -294,31 +294,14 @@ jobs:
           test/test_cvedb.py
       - name: Run HTML tests
         run: pytest -v -n auto test/test_html.py
-      - name: Cache conda
-        uses: actions/cache@v3
-        env:
-          # Increase to reset cache if requirements.txt file has not changed
-          CACHE_NUMBER: 0
-        with:
-          path: ~/conda_pkgs_dir
-          key: ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{
-            hashFiles('requirements.txt') }}
-      - uses: conda-incubator/setup-miniconda@v2
-        with:
-          auto-update-conda: true
-          activate-environment: pdftotext
-          use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
-      - name: Install pdftotext, reportlab and cve-bin-tool
+      - name: Install reportlab and cve-bin-tool
         run: |
-          conda install -c conda-forge python=3.10 poppler pdftotext
           python -m pip install --upgrade pip
           python -m pip install --upgrade setuptools
           python -m pip install --upgrade wheel
           python -m pip install --upgrade reportlab
           python -m pip install --upgrade -r dev-requirements.txt
           python -m pip install --upgrade .
-      - name: Test PDF generation on Windows
```

Comment:
You can actually remove these pip install lines too.  They're already done above before the tests are run, and I'm not sure if they're duplicated down here for a reason (maybe something about matching versions with conda?) but they're definitely not needed if we're not running anything after them.

Reply:
In case it's not clear, that's the pip install lines and the "-name: Install..." and "run" lines above too.

Reply:
Should be good now 👍🏻